libraries/GCS_MAVLink/GCS.h

```diff
@@ -323,6 +324,8 @@ class GCS_MAVLINK
     // this allows us to detect the user wanting the CLI to start
     uint8_t        crlf_count;
 
+    static bool _in_update;
```

Comment:
Making this static will mean backends which might have been able to get work done will not be able to.